modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecCommandRoutes.scala

```diff
@@ -21,18 +25,18 @@ import scalaz.concurrent.Task
   */
 class SeqexecCommandRoutes(auth: AuthenticationService, inputQueue: engine.EventQueue, se: SeqexecEngine) extends BooEncoders {
 
-  val tokenAuthService = JwtAuthentication(auth, auth.config.devMode)
-
   val commands = Commands(se.odbProxy)
+  // Handles authentication
+  val httpAuthentication = new Http4sAuthentication(auth)
 
-  val service = tokenAuthService { GZip { HttpService {
-    case GET  -> Root  / "host" =>
+  val commandServices: AuthedService[UserDetails] = AuthedService {
+    case GET  -> Root  / "host" as _ =>
```

Comment:
that is an extractor that lets you read the user data doing `as user`. However in this case we don't care who the user is just that we have one so we can use the underscore